pom.xml

```diff
@@ -45,6 +45,7 @@
     </scm>
 
     <modules>
+        <module>annotations</module>
```

Comment:
Just a nitpick, but all the other modules are non-plural (constraint, expression, etc.)  They probably should be consistent.

Reply:
Yeah, I was also wondering about singular or plural. I've ended up using plural because other annotation based projects use the plural version: `jboss-annotations`, `hibernate-annotations`, etc.